bin/p2-preparer/main.go

```diff
@@ -34,6 +34,12 @@ func main() {
 		logger.WithError(err).Fatalln("invalid parameter")
 	}
 
+	if preparerConfig.ReadyFile != "" {
+		if _, err := os.Stat(preparerConfig.ReadyFile); os.IsNotExist(err) {
+			logger.WithError(err).Fatalln("ready file not present")
```

Comment:
🤔 

I don't think we wanted the **preparer** to fail to start though. I think we wanted to make the preparer read from this config to tell p2-exec what file to check